### DISCUSSION TITLE
Accessing the SOC From Behind a firewall/NAT

### DISCUSSION THREAD

**Discussion by vector1one** [Security-Onion-Solutions/securityonion#5635] [Unsupported Versions]
current set up

laptop(192.168.0.1 ) >   firewall [(NAT/Port FRWD )192.168.1.1:1234 ] > (ESXI)(SO2-SOC)192.168.1.2
        
I have set up port forwarding with pfsense in a ESXI and have used so-allow for the out facing ip, the port forward works and the SOC page connects but its just blank. the port forward is working, but something on the SO-SOC side is preventing.

is there anyway to have the SOC page associated with a port number to allow forwarding ?

**Comment by dougburks** [comment]
We allow you to change the IP address or URL that you are using to access SOC, but I'm not sure this allows you to change the actual port number:
https://docs.securityonion.net/en/2.3/url-base.html
